collection-scripts/gather_ppc

```diff
@@ -45,8 +45,12 @@ function ppc_nodes() {
   NTO=$(oc get deployment -n openshift-cluster-node-tuning-operator cluster-node-tuning-operator -o jsonpath="{.spec.template.spec.containers[0].image}")
   if [ $? -ne 0 ]; then
     echo "ERROR: Failed to identify the container image with node tools."
-    echo "INFO: Node performance data collection will not contain node level data."
-    return
+    echo "INFO: Fallback to identify the container image from release info"
+    NTO=$(oc adm release info --image-for=cluster-node-tuning-operator)
+    if [ $? -ne 0 ]; then
+      echo "INFO: Node performance data collection will not contain node level data."
```

Comment:
You should probably log the second error here as well so we can tell from the logs which method passed / failed.

Reply:
Or just move the ERROR here

Reply:
Done, moved the error inside the nested `if`